static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-dismiss-window.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+export const signInGateDismissWindow: ABTest = {
+    id: 'SignInGateDismissWindow',
+    start: '2020-08-12',
+    expiry: '2020-12-01',
+    author: 'vlbee',
+    description:
+        'Dismiss Window sign in gate test on 3nd article view of simple article templates, with higher priority over banners and epic',
+    audience: 0.0244,
+    audienceOffset: 0.8855,
```

Comment:
I think it might be better to round the audience to `0.025` and the offset to `0.875`.

In my opinion, this makes the audience size easier to work with.

Using this offset will also mean that it doesn't overlap into the control audience, who should never see a gate.